example-app/SantokuApp/src/features/qa-event/components/EventList.tsx

```diff
@@ -18,25 +19,37 @@ export type EventListProps = {
 
 export const EventList: React.FC<EventListProps> = ({data, containerWidth}) => {
   const theme = useTheme<RestyleTheme>();
-  const eventListCardWidth = useMemo(() => containerWidth - theme.spacing.p24 * 2, [containerWidth, theme.spacing.p24]);
+  const eventListCardWidth = useMemo(() => {
+    // カード幅は、画面幅からカードのはみ出し部分とカード間の余白を引いた値である。
+    // カードのはみ出し部は8px、カード間の余白は16pxであり、それぞれカードの左右に存在することを考慮すると
+    // カード幅の算出式は以下のようになる。
+    return containerWidth - (theme.spacing.p8 + theme.spacing.p16) * 2;
+  }, [containerWidth, theme.spacing.p16, theme.spacing.p8]);
   return (
-    <StyledScrollView horizontal showsHorizontalScrollIndicator={false} pagingEnabled>
+    <StyledScrollView
+      horizontal
+      showsHorizontalScrollIndicator={false}
+      snapToInterval={Platform.select({ios: containerWidth - 32})}
+      snapToAlignment={Platform.select({android: 'center'})}
+      pagingEnabled={Platform.select({android: true})}
+      decelerationRate="fast">
+      <StyledSpace width="p16" />
       {data?.map(item => (
         <StyledRow key={item.eventId}>
-          <StyledSpace width="p24" />
+          <StyledSpace width="p8" />
           <EventListCard event={item} containerWidth={eventListCardWidth} />
-          <StyledSpace width="p24" />
+          <StyledSpace width="p8" />
         </StyledRow>
       ))}
-      <StyledSpace width="p24" />
+      <StyledSpace width="p8" />
       <Box width={eventListCardWidth} justifyContent="center" alignItems="center">
         <StyledTouchableOpacity onPress={showUnderDevelopment}>
           <Text variant="font14Bold" lineHeight={20} letterSpacing={0.25} color="blue">
             {m('もっと見る')}
           </Text>
         </StyledTouchableOpacity>
       </Box>
-      <StyledSpace width="p24" />
+      <StyledSpace width="p8" />
```

Comment:
最後の要素は 幅 24 の余白を右側に作らないと中心がずれます。
```suggestion
      <StyledSpace width="p24" />
```

↓元の `p8` 指定の一番右のカードの領域（背景指定をしてわかりやすくしています）
<img width="453" alt="スクリーンショット 2023-03-27 16 31 57" src="https://user-images.githubusercontent.com/13145001/227871642-09d42625-f0f4-4730-8d94-2750171ab744.png">